examples/CQL/README.md

```diff
@@ -0,0 +1,35 @@
+## Reproduce CQL with PARL
+
+Different from Online RL which gather new data directly from the environment, Offline RL algorithms learn a policy or Q-function from static dataset, without additional interaction with the environment.
```

Comment:
Remove the sentence about the offline RL. We would like the reader to refer to the original paper for more introduction.